### ISSUE TITLE
wrong credentials for rabbitmq aren't shown as error

### ISSUE THREAD

**Issue by phwoelfel** [dgray10/SecPi#93]
on manager:

```
2016-06-21 10:27:36,738 [INFO] Initializing manager
2016-06-21 10:27:39,969 [ERROR] Wasn't able to connect to rabbitmq service: 
2016-06-21 10:28:13,117 [ERROR] Wasn't able to connect to rabbitmq service: 
```

rabbitmq log:

```
=ERROR REPORT==== 21-Jun-2016::10:28:13 ===
closing AMQP connection <0.27394.1> (10.0.0.8:62013 -> 10.0.0.201:5671):
{handshake_error,starting,0,
                 {amqp_error,access_refused,
                             "PLAIN login refused: user 'secpi' - invalid credentials",
                             'connection.start_ok'}}
```

**Comment by MartinLiebl** [comment]
Now the error message if the username or password is wrong is:
"Wasn't able to connect to rabbitmq service: Client was disconnected at a connection stage indicating a probable authentication error"

When the rabbitmq service is not running the error message shown is:
"Wasn't able to connect to rabbitmq service: The AMQP connection was closed: ()"

For the first error the message seems to be ok, but for the second error I think the message is not completely clear about the situation.

**Comment by MartinLiebl** [comment]
Fixed webinterface part with commit feb02f2.
